mappings/net/minecraft/block/ChainBlock.mapping

```diff
@@ -1,3 +1,3 @@
 CLASS net/minecraft/class_5172 net/minecraft/block/ChainBlock
-	FIELD field_23986 SHAPE Lnet/minecraft/class_265;
+	FIELD field_23986 Y_SHAPE Lnet/minecraft/class_265;
```

Comment:
This is not applicable as chains only have one direction (on the Y axis) in 1.16.1.